src/restClient.js

```diff
@@ -8,23 +8,32 @@ import {
   DELETE,
 } from 'admin-on-rest/lib/rest/types';
 
-export default client => {
+export default (client, options = { id: 'id' }) => {
   const mapRequest = (type, resource, params) => {
     const service = client.service(resource);
     let query = {};
 
+    const mapParamsId = params => {
+      if(options.id !== 'id'){
+        params[options.id] = params.id;
+        delete params.id;
+      }
+      return params;
```

Comment:
Tests pass (but here's why):
https://github.com/josx/aor-feathers-client/blob/master/test/restClient.spec.js#L154

For update the test we have to test it with different Ids keys. So we must:

* Alter all json strucutres to support dynamic id key
* Test the usecase for default id key param
* Repeat tests with another id key (fakeId)